mmv1/third_party/terraform/services/gkehub2/resource_gke_hub_fleet_test.go.erb

```diff
@@ -15,6 +15,147 @@ import (
 	transport_tpg "github.com/hashicorp/terraform-provider-google/google/transport"
 )
 
+func TestAccGKEHub2Fleet_gkehubFleetWithComplianceConfig_update(t *testing.T) {
+	t.Parallel()
+
+	context := map[string]interface{}{
+		"random_suffix":   acctest.RandString(t, 10),
+		"org_id":          envvar.GetTestOrgFromEnv(t),
+		"billing_account": envvar.GetTestBillingAccountFromEnv(t),
+	}
+
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
+		CheckDestroy:             testAccCheckGKEHub2FleetDestroyProducer(t),
+		ExternalProviders: map[string]resource.ExternalProvider{
+			"time":   {},
+		},
+		Steps: []resource.TestStep{
+			{
+				Config: testAccContainerCluster_EnableComplianceConfigWithDefaultStandards(clusterName, networkName, subnetworkName),
```

Comment:
```suggestion
				Config: testAccContainerCluster_EnableComplianceConfigWithDefaultStandards(context),
```

Tests and checks failed due to error like
```
Error: google-beta/services/gkehub2/resource_gke_hub_fleet_test.go:37:80: undefined: clusterName
Error: google-beta/services/gkehub2/resource_gke_hub_fleet_test.go:37:93: undefined: networkName
Error: google-beta/services/gkehub2/resource_gke_hub_fleet_test.go:37:93: too many arguments in call to testAccContainerCluster_EnableComplianceConfigWithDefaultStandards
```

We may also need to update the rest of tests as well